public/app/common/components/cc-form/cc-form.jsx

```diff
@@ -0,0 +1,36 @@
+var React = require('react');
+var styleMixin = require('mixins/style-mixin');
+var t = require('tcomb-form');
```

Comment:
We could use tForms instead, but we need somehow separate out control center form, library form and etc.